rose/server/web/index.html

```diff
@@ -9,11 +9,20 @@
 
   <body>
     <div id="content">
+      <a href="settings.html" id="settings-link">Settings</a>
```

Comment:
you might want to make this `settings/` and rename settings.html to settings/index.html

Reply:
@bennypowers i currently put the web component in `settings-form` file in the same dir, is there a convention for how to name web components ? where to put tehm ?

Reply:
Usually, name the module the same as the tag name